### PR TITLE
Adding the create starknet account logic for Kotlin

### PR DESCRIPTION
**Pull request by Gerson2102**
- [#67] issue #
- [X] follows contribution [guide](https://github.com/keep-starknet-strange/starknet-phone/blob/main/CONTRIBUTING.md)
- [ ] code change includes tests
- [ ] breaking change

Adding the create starknet account logic for Kotlin. I have an error currently. I will give more details inside of the PR.

### CODE REVIEW

**Comment by Gerson2102** [comment]
This is the error, it seems related to the RPC. But I'm not sure:
image

**Comment by trbutler4** [comment]
> This is the error, it seems related to the RPC. But I'm not sure: image
> 
> The RPC that I'm using is katana.

Android is blocking http. You can try using a provider like alchemy, which will give you a free https starknet rpc for the time being. Alternatively you can add the katana url as an allowed url by following option 2b in this guide: https://www.geeksforgeeks.org/android-cleartext-http-traffic-not-permitted/

**Comment by Gerson2102** [comment]
> > This is the error, it seems related to the RPC. But I'm not sure: image
> > The RPC that I'm using is katana.
> 
> Android is blocking http. You can try using a provider like alchemy, which will give you a free https starknet rpc for the time being. Alternatively you can add the katana url as an allowed url by following option 2b in this guide: https://www.geeksforgeeks.org/android-cleartext-http-traffic-not-permitted/ , or just allow all http traffic with option 1b. However we dont want to commit 1b.

Got it. I already did all that stuff. But still not working. Maybe I should change the RPC, do you have that RPC from Alchemy or how can I get that?

**Comment by Gerson2102** [comment]
Now its working:
image
I just put a different address, from `0.0.0.0` to `10.0.2.2`.

**Comment by trbutler4** [comment]
@Gerson2102 Looks good, can you take a look at the conflicts please?

File: wallet_app/android/app/build.gradle.kts [hunk 1]
```diff
@@ -65,6 +65,10 @@ android {
 }
 dependencies {
 
+    implementation("com.swmansion.starknet:starknet:0.12.1@aar"){
```
Comment (trbutler4) [inline wallet_app/android/app/build.gradle.kts:68]: this is a duplicate dependency. Either remove this one, or the other dep on line 100

Comment (trbutler4) [inline wallet_app/android/app/build.gradle.kts:68]: this is a duplicate, either remove this or line 100.

**Review by trbutler4 — CHANGES_REQUESTED** [review]

**Comment by trbutler4** [comment]
Other than change i requested this is a good start. Will merge once that is addressed

**Comment by trbutler4** [comment]
actually i will just merge and then remove it.

**Comment by Gerson2102** [comment]
Thanks, yesterday I was deleting that dependency por for a reason I couldn't run the app again. And I was trying again today and same story.

**Pull request merged**